projects/fal/pyproject.toml

```diff
@@ -14,7 +14,6 @@ python = "^3.8"
 isolate = {version = ">=0.12.3, <1.0", extras = ["build"]}
 isolate-proto = {path = "../isolate_proto", develop = true}
 auth0-python = "^4.1.0"
```

Comment:
We only seem to use this to re-validate the token we receive, is that simply a debug assertion @chamini2? Can we drop `validate_id_token` and with it, `auth0` dependency?

Reply:
looking into it